ci/verify/release-pipeline.yaml

```diff
@@ -95,34 +100,63 @@ spec:
       - put: vmc-image
         params: &image_put_params
           image: image/image.tar
-          additional_tags: src/.git/short_ref
+          additional_tags: vmc-src/.git/short_ref
 
-    - name: release
+    - name: build-cloudhsm-client
       serial: true
       plan:
+      - get: cloudhsm-client-src
+        trigger: true
+      - task: build-cloudhsm
+        privileged: true
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: vito/oci-build-task
+            version:
+              digest: sha256:cfb2983956145f54a4996c2aff5fc598856c8722922a6e73f9ebfa3d9b3f9813
+          params:
+            CONTEXT: cloudhsm-client-src/cloudhsm
+          inputs:
+          - name: cloudhsm-client-src
+          outputs:
+          - name: image
```

Comment:
Let's avoid using the same output name in the same job for different things - someone could come along and try to run these builds in_parallel

Reply:
disagree ... these task configs should be reusable ... it should be possible to extract them into a task file and using input_mapping / output_mapping to move inputs